frontend/src/component/Account/UserSpacesConfigLine.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+
+import {
+  BtnSwitch,
+  IconButton,
+  ROLE_LIST
+} from 'tracim_frontend_lib'
+
+class UserSpacesConfigLine extends React.Component {
+  render () {
+    const { props } = this
+    const { space, member } = props
+    const memberRole = ROLE_LIST.find(r => r.slug === member.role)
+    return (
+      <tr>
+        <td>
+          <div className='spaceconfig__table__spacename'>
+            {space.label}
+          </div>
+        </td>
+
+        <td>
+          <div className='spaceconfig__table__role'>
+            <div className='spaceconfig__table__role__icon'>
+              <i className={`fa fa-fw fa-${memberRole.faIcon}`} style={{ color: memberRole.hexcolor }} />
+            </div>
+            <div className='spaceconfig__table__role__text d-none d-sm-flex'>
+              {props.t(memberRole.label)}
+            </div>
+          </div>
+        </td>
+
+        <td>
+          <BtnSwitch
+            checked={member.do_notify}
+            onChange={() => props.onChangeSubscriptionNotif(space.workspace_id, !member.do_notify)}
+          />
+        </td>
+        <td data-cy='spaceconfig__table__leave_space_cell'>
+          <IconButton
+            className='outlineTextBtn primaryColorBorder primaryColorBgHover primaryColorBorderDarkenHover'
+            onClick={(() => props.onLeaveSpace(space.workspace_id))}
+            icon='sign-out'
+            text={props.admin ? props.t('Remove from space') : props.t('Leave space')}
+          />
```

Comment:
As you'll merge after my PR, be aware that `IconButton` changed a little.
To have the same button as this one you will have to declare it as:
```javascript
<IconButton
  mode="light"
  onClick={(() => props.onLeaveSpace(space.workspace_id))}
  icon='sign-out'
  text={props.admin ? props.t('Remove from space') : props.t('Leave space')}
/>
```

Reply:
thx